### PR TITLE
Add debugging info

### PR DESCRIPTION
**Pull request by keyur32**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809211838491162-394%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jdIQGU%2FShvImSFQnK7SDYDUQetl%2BHVvU%2FgpVS%2Ffkq6g%3D&st=2018-09-21T18%3A35%3A01Z&se=2018-10-22T18%3A40%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-best-practices.md [hunk 1]
```diff
@@ -41,7 +41,11 @@ You can enable error logging for your custom functions add-in in multiple ways,
 
 ## Debugging
 
-Currently, the best method for debugging Excel custom functions is to use [Excel Online](https://www.office.com/launch/excel) and use the F12 debugging tool native to your browser. Additional debugging tools for custom functions may be available in the future.
+Currently, the best method for debugging Excel custom functions is to first [sideload](https://docs.microsoft.com/en-us/office/dev/add-ins/testing/sideload-office-add-ins-for-testing) them on Excel Online. Then you can use the [F12 debugging tool native to your browser](https://docs.microsoft.com/en-us/office/dev/add-ins/testing/debug-add-ins-in-office-online).
+
+If your add-in fails to register, please verify your SSL certificates are correctly installed.
```
Comment (mscharlock) [inline docs/excel/custom-functions-best-practices.md]: Add SSL site link: https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md

**Comment by mscharlock** [comment]
Sorry, meant to not approve

File: docs/excel/custom-functions-best-practices.md [hunk 2]
```diff
@@ -41,7 +41,11 @@ You can enable error logging for your custom functions add-in in multiple ways,
 
 ## Debugging
 
-Currently, the best method for debugging Excel custom functions is to use [Excel Online](https://www.office.com/launch/excel) and use the F12 debugging tool native to your browser. Additional debugging tools for custom functions may be available in the future.
+Currently, the best method for debugging Excel custom functions is to first [sideload](https://docs.microsoft.com/en-us/office/dev/add-ins/testing/sideload-office-add-ins-for-testing) them on Excel Online. Then you can use the [F12 debugging tool native to your browser](https://docs.microsoft.com/en-us/office/dev/add-ins/testing/debug-add-ins-in-office-online).
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/excel/custom-functions-best-practices.md]: also, suggest these wording changes: 

- change the phrase "sideload them on Excel Online" to "sideload your add-in within Excel Online"
- change the second sentence to: "Then you can debug your custom functions by using the [F12 debugging tool native to your browser](../testing/debug-add-ins-in-office-online).

File: docs/excel/custom-functions-best-practices.md (hunk 1 above)
Comment (kbrandl) [inline docs/excel/custom-functions-best-practices.md]: Suggest changing "...verify your SSL certificates are correctly installed."  to "...verify that SSL certificates are correctly configured for the web server that's hosting your add-in application."

**Comment by o365devx** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-best-practices.md) | :warning:Warning | | [Details](#user-content-docs%2Fexcel%2Fcustom-functions-best-practices.md)
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

<a name="docs%2Fexcel%2Fcustom-functions-best-practices.md"></a>
### [docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-best-practices.md)
 - **Line 44**: **[Warning]** Invalid file link:(~/testing/debug-add-ins-in-office-online).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809220110138561-394%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vhJPwvxYOQBXb5f17Ye5Sa451jfx9oac0CuJJyXcfx8%3D&st=2018-09-22T01%3A06%3A29Z&se=2018-10-23T01%3A11%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809220113380815-394%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zcgi8ZUxRJ9bf0IvARyivohpG5RSkU3diwwi95cbmYQ%3D&st=2018-09-22T01%3A09%3A58Z&se=2018-10-23T01%3A14%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809232300022464-394%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fP2J8g%2FryAjX6nGDAFOgkDJkfn1TASZGMLFyMkT4jvQ%3D&st=2018-09-23T22%3A56%3A18Z&se=2018-10-24T23%3A01%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**